File: sandbox/server/CodeVerse/CodeVerse.BlogsDemo/CodeVerse.BlogsDemo.API/Models/Entity/Blog.cs
namespace CodeVerse.BlogsDemo.API.Models.Entity
{
    public class Blog
    {
        public Blog()
        {
            Posts = new List<Post>();
        }

        public long Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public virtual List<Post> Posts { get; set; }
    }
}

File: sandbox/client/DemoApp/app/src/main/java/com/example/demoapp/models/Blog.kt
package com.example.demoapp.models

data class Blog(

    val id: Long,
    val title : String? = null,
    val posts : MutableList<Post>
)
//    val userId : Int? = null,
//    val id: String? = null,
//    val title : String? = null,
//    val body : String? = null)